backend/Dockerfile

```diff
@@ -3,7 +3,7 @@
 
 FROM --platform=linux/amd64 public.ecr.aws/lambda/python:3.9
 
-ADD backend backend
 
 COPY requirements.txt  .
 RUN  pip3 install -r requirements.txt --target "${LAMBDA_TASK_ROOT}"
+ADD backend backend
```

Comment:
I am curious if this change is intentional.

Reply:
Yes, this makes it so that we don't have to run the pip install step every time a file changes in the backend folder. Speeds up deploys a lot. If requirements.txt changes, the pip install will run.